pkg/cmd/hub/operator.go

```diff
@@ -22,6 +22,10 @@ func NewHubOperatorCmd() *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.BoolVar(&cmOptions.SkipRemoveCRDs, "skip-remove-crds", false, "Skip removing CRDs while ClusterManager is deleting.")
+	flags.StringToStringVar(&cmOptions.MasterNodeLabelSelector, "master-node-label-selector",
```

Comment:
The flag is named `label-selector`, so I think it should have 2 parts : MatchLabels and MatchExpressions, like the definition of LabelSelector  https://github.com/open-cluster-management-io/ocm/blob/92a2c91f15947d30627bf2652c5155cf81971437/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L1213  So I think string type is better than map here.
Otherwise maybe we need to rename the flag name, for example  control-plane-node labels ?

Reply:
renamed to `control-plane-node-labels`